types/react-native-sensor-manager/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for react-native-sensor-manager 0.1
+// Project: https://github.com/kprimice/react-native-sensor-manager
+// Definitions by: Sahin Vardar <https://github.com/SahinVardar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="react-native" />
+
+export * from 'react-native'
+
+interface SensorManager {
+    startAccelerometer(delay: number): void;
+    stopAccelerometer(): void;
+    startGyroscope(delay: number): void;
+    stopGyroscope(): void;
+    startMagnetometer(delay: number): void;
+    stopMagnetometer(): void;
+    startStepCounter(delay: number): void;
+    stopStepCounter(): void;
+    startThermometer(delay: number): void;
+    stopThermometer(): void;
+    startMotionValue(delay: number): void;
+    stopMotionValue(): void;
+    startOrientation(delay: number): void;
+    stopOrientation(): void;
+    startProximity(delay: number): void;
+    stopProximity(): void;
+    startLightSensor(delay: number): void;
+    stopLightSensor(): void;
+}
+
+declare module 'react-native' {
+    export interface DeviceEventEmitterStatic {
```

Comment:
Btw looking at the original definitions my augmented functions have a return void type instand of EmitterSubscription, I will fix that to be consistent here.